inc/cachify_hdd.class.php

```diff
@@ -303,19 +303,16 @@ public static function _dir_size( $dir = '.' ) {
 	private static function _file_path( $path = null ) {
 		$prefix = is_ssl() ? 'https-' : '';
 
+		$host_parts = wp_parse_url( 'http://' . strtolower( $_SERVER['HTTP_HOST'] ) );
```

Comment:
Absolutely right. My commit didn't optimize any existing code ;)

If we're talking about tweaks, `cachify.class.php` lines 1045 ff. could also be simplified, as `wp_parse_url( '' ) ['path'] === ''`, so the `empty()` check is redundant...